modules/virt-importing-vm-wizard.adoc

```diff
@@ -53,42 +84,28 @@ The CA certificate is the second certificate in the output.
 ** *Password*: RHV Manager password
 
 * If you select a saved RHV instance, the wizard connects to the RHV instance using the saved credentials.
-endif::[]
 
 . Click *Check and Save* and wait for the connection to complete.
-ifdef::virt-importing-rhv-vm[]
 +
 [NOTE]
 ====
 The connection details are stored in a secret. If you add a provider with an incorrect URL, user name, or password, click *Workloads* -> *Secrets* and delete the provider secret.
 ====
 
-. Select a cluster and a virtual machine to import.
-endif::[]
-ifdef::virt-importing-vmware-vm[]
-+
-[NOTE]
-====
-The connection details are stored in a secret. If you add a provider with an incorrect host name, user name, or password, click *Workloads* -> *Secrets* and delete the provider secret.
-====
-
-. Select a virtual machine or a template to import.
-ifeval::["{VirtVersion}" < "2.5"]
-+
-[IMPORTANT]
-====
-The VMware VM name must not contain a forward slash (`/`).
-====
+. Select a cluster and a virtual machine.
 endif::[]
+. Click *Next*.
+. In the *Review* screen, review your settings.
```

Comment:
In the output, I see what looks to be two duplicated steps:

7. Click Next.
8. In the Review screen, review your settings.
9. Click Next.
10. In the Review screen, review your settings.

This is for VMware. RHV looks fine.